workspace_run_task.go

```diff
@@ -60,12 +61,16 @@ type WorkspaceRunTaskCreateOptions struct {
 	EnforcementLevel TaskEnforcementLevel `jsonapi:"attr,enforcement-level"`
 	// Required: The run task to attach to the workspace
 	RunTask *RunTask `jsonapi:"relation,task"`
+	// Optional: The stage to run the task in. This is currently in BETA
+	Stage *Stage `jsonapi:"attr,stage,omitempty"`
 }
 
 // WorkspaceRunTaskUpdateOptions represent the set of options for updating a workspace run task.
 type WorkspaceRunTaskUpdateOptions struct {
 	Type             string               `jsonapi:"primary,workspace-tasks"`
 	EnforcementLevel TaskEnforcementLevel `jsonapi:"attr,enforcement-level,omitempty"`
+	// The stage to run the task in. This is currently in BETA
+	Stage Stage `jsonapi:"attr,stage,omitempty"`
```

Comment:
Looks like the `WorkspaceRunTaskCreateOptions`, expects a `Stage pointer`, but the `WorkspaceRunTaskUpdateOptions` takes the `Stage value`, is there a reason for the difference or should both take same construct for consistency?

Reply:
Hrmm yes... I was copying the pattern of  enforcement level.  Technically that should also be a pointer.

Reply:
Fixed